content/id/docs/concepts/containers/container-environment-variables.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Variabel Environment Kontainer
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan berbagai *resource* yang tersedia di dalam Kontainer pada suatu *environment*.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## *Environment* Kontainer
+
+*Environment* Kontainer pada Kubernetes menyediakan beberapa *resource* penting yang tersedia di dalam Kontainer:
+
+* Sebuah *Filesystem*, yang merupakan kombinasi antara [image](/docs/concepts/containers/images/) dan satu atau banyak [*volumes*](/docs/concepts/storage/volumes/).
+* Informasi tentang Kontainer tersebut.
+* Informasi tentang objek-objek lain di dalam kluster.
+
+### Informasi tentang Kontainer
+
+*Hostname* sebuah Kontainer merupakan nama dari Pod dimana Kontainer dijalankan.
+Informasi ini tersedia melalui perintah `hostname` atau panggilan (*function call*) 
+[`gethostname`](http://man7.org/linux/man-pages/man2/gethostname.2.html) pada libc.
+
+Nama Pod dan *namespace* tersedia sebagai variabel *environment* melalui [API *downward*](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/).
+
+Variabel *environment* yang ditulis pengguna dalam Pod *definition* juga tersedia di dalam Kontainer,
+seperti halnya variabel *environment* yang ditentukan secara statis di dalam *image* Docker.
+
+### Informasi tentang Kluster
+
+Daftar semua *Service* yang dijalankan ketika suatu Kontainer dibuat, tersedia di dalam Kontainer tersebut sebagai variabel *environment*.
+Variabel-variabel *environment* tersebut sesuai dengan sintaksis *links* dari Docker.
```

Comment:
I think we should put "sintaksis" inside the glossary as well. It's not really common in Bahasa

Reply:
sure, it's there in the sheet